unit_tests/test_version_utils.py

```diff
@@ -564,3 +565,28 @@ def test_comparable_scylla_operator_versions_compare(version_string_left, versio
 ))
 def test_comparable_scylla_operator_versions_to_str(version_string_input, version_string_output):
     assert str(ComparableScyllaOperatorVersion(version_string_input)) == version_string_output
+
+
+@pytest.mark.parametrize("scylla_version,distro,expected_repo", (
```

Comment:
Let's mark it as integration tests, since it's hitting s3

Reply:
done